eips_code/eip3540.py

```diff
@@ -9,45 +9,57 @@
 def is_eof(code: bytes) -> bool:
     return code.startswith(MAGIC)
 
+class ValidationException(Exception):
+    pass
 
 # Validate EOF code.
 def validate_eof(code: bytes):
```

Comment:
Could have left the exception comment here as in other files.

Reply:
fixed